Formula/lp_solve.rb

```diff
@@ -4,6 +4,7 @@ class LpSolve < Formula
   url "https://downloads.sourceforge.net/lpsolve/lp_solve_5.5.2.11_source.tar.gz"
   sha256 "6d4abff5cc6aaa933ae8e6c17a226df0fc0b671c438f69715d41d09fe81f902f"
   license "LGPL-2.1-or-later"
+  revision 1
```

Comment:
The changes you've made look like a no-op on macOS. If they are, then the revision bump is unnecessary, no?

Reply:
I figured it would help with keeping `revision` in sync between macOS/Linux for now. Not sure if it's something to be concerned about (could certainly ignore the mismatch).

Reply:
It isn't bottled on linux, so this isn't really needed I think.

Reply:
Did not notice that, thanks for catching it.